monitoring/statistics.h

```diff
@@ -53,6 +53,12 @@ class StatisticsImpl : public Statistics {
   virtual void setTickerCount(uint32_t ticker_type, uint64_t count) override;
   virtual uint64_t getAndResetTickerCount(uint32_t ticker_type) override;
   virtual void recordTick(uint32_t ticker_type, uint64_t count) override;
+  // Still get it implemented for now, in case a user explictly called it.
+  // For example, they may have a wrapped Statistics object, passing the call
+  // to recordTick() into here. We'll eventually remove the assert.
+  virtual void measureTime(uint32_t histogramType, uint64_t time) {
```

Comment:
According to fbcode, we should favor `void measureTime(uint32_t histogramType, uint64_t time) override` from now on.

Reply:
We should change fbcode's implementation of Statistics instead...